lib/init/parser.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const parser = require('body-parser');
```

Comment:
See if you can replace this with `connect-multiparty` so we don't have two body parsers